pkg/auth/internal/tokenstore/tokenstore.go

```diff
@@ -7,10 +7,17 @@ import (
 	"go.jetpack.io/pkg/auth/session"
 )
 
+const storeDataVersion = "1"
+
 type Store struct {
 	rootDir string
 }
 
+type storeData struct {
+	Version string           `json:"version"`
+	Tokens  []*session.Token `json:"tokens"`
```

Comment:
Would be nice to comment here that the order matters: tokens in front are given priority, and a new token is pre-pended to the existing list